buf/registry/module/v1/module_service.proto

```diff
@@ -118,7 +118,8 @@ message CreateModulesRequest {
     // The module's visibility.
     ModuleVisibility visibility = 3 [
       (buf.validate.field).required = true,
-      (buf.validate.field).enum.defined_only = true
+      (buf.validate.field).enum.defined_only = true,
+      (buf.validate.field).enum.not_in = 0
```

Comment:
Was chatting with @rodaine about this w.r.t. my PR, and this not_in is redundant with required. Using required is probably better UX so propose removing this here.
```suggestion
```